### DISCUSSION TITLE
W610 supported via wifi?

### DISCUSSION THREAD

**Discussion by sbonera** [nathanmarlor/foxess_modbus#478] [Support] [answered]
I have been running w610 on wifi to the modbus on the inverter.  I think i have sorted my configuration yaml (had an early version of stealthCheshunt so i had to change template and sensor config) . 

A couple of questions if I may

As I am migrating will it only support if I plug ethernet into the w610.
The version of stealthCheshunt I had had no unique ID's will this be a problem for this integration
Any one kind enough to check my configuration yaml.  😄 

Thanks to everyone contributing to this great HA addon.

**Comment by canton7** [comment, marked as answer]
We can connect to the W610 over WiFi just fine - HA doesn't even know whether you're using wireless or wired.

I *think* the unique id situation is fine, as we use the same entity ids, but will let @FozzieUK confirm.

**Comment by FozzieUK** [comment]
Hi @sbonera , it’s no problem if you don’t have unique id’s (it actually causes more potential problems if you do have them)

I’d be happy to check over your configuration.yaml for you 😊

**Reply by sbonera** [reply]
@FozzieUK thanks will reach out to you for the config check.

**Reply by FozzieUK** [reply]
Ok, no problem 👍
